resources/splunk_install.rb

```diff
@@ -60,25 +50,43 @@ def package_path
 
   load_current_value do |desired|
     package desired.package
-    version_file = version_pathname(install_dir)
-    current_value_does_not_exist! unless version_file.exist?
 
-    version_data = version_file.read
-    installed_version, installed_build = version_data.match(/VERSION=(\d(?:\.\d)+).+BUILD=([\w]+)/m).captures
-    version installed_version
-    build installed_build
+    user_resource = Chef::Resource::User.new(desired.user || new_resource.user, run_context)
+    user_resource.system true
+    user_resource.manage_home true
+    user_resource.run_action :create
```

Comment:
Should we be creating a user in at the point where we're "loading the current values" ?